.github/workflows/release-post-install-hook.yml

```diff
@@ -0,0 +1,57 @@
+name: Release Post Install Hook
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Release version:"
+        required: true
+
+jobs:
+  release-post-install-hook:
+    name: Release Post Install Hook
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.inputs.version }}
+      QUAY_RELEASE_REPO: mongodb/atlas-operator-post-install-hook
+      QUAY_ROBOT_NAME: mongodb+mongodb_atlas_kubernetes
+      QUAY_RELEASE_REGISTRY: quay.io
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
+
+      - name: Install axios
+        run: npm install axios
+
+      - name: Verify release version already exists
+        id: verify-release-version
+        uses: actions/github-script@v5
+        env:
+          image: atlas-operator-post-install-hook
+          version: ${{ github.event.inputs.version }}
+        with:
+          result-encoding: string
+          script: |
+            const axios = require("axios");
+            const {image, version} = process.env;
+            const resp = await axios.get(`https://quay.io/api/v1/repository/mongodb/${image}`);
+            return resp.data.tags[version] !== undefined;
+
+      - name: Fail if release exists
+        if: steps.verify-release-version.result != 'false'
+        env:
+          version: ${{ github.event.inputs.version }}
+        run: |
+          echo "Release ${version} already exists"
+          exit 1
+
+      - name: Push Post Install Hook To Registry
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
probably QUAY_ROBOT_NAME

Reply:
I don't see this username in our secrets 🤔

Reply:
ah I see you mean from same workflow, not secrets!